packages/manager/modules/pci/src/projects/project/private-networks/add/add.controller.js

```diff
@@ -79,6 +79,7 @@ export default class NetworkAddCtrl {
   isVlanAvailable(vlanId) {
     return !find(this.networks, { vlanId });
   }
+
```

Comment:
Seems not related. Revert-it to avoid a new release of pci module.

Reply:
I have tried to remove it but it seems that eslint fix it automatically

Reply:
Fixed by #978